Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM gcr.io/cloud-solutions-images/jenkins-k8s-slave
+
+USER root
+
+# Expose Sauce Connect port
+EXPOSE 4445
+
+RUN apt-get update
+
+# wpt run dependencies
+RUN apt-get install -y python-pip
+RUN pip install virtualenv
+RUN pip install 'requests==2.13.0'
+
+RUN apt-get install -y git
+
+# Needed for hosts_fixup
+RUN apt-get install -y sudo
+
+# Used for running FF and Chrome
+RUN apt-get install -y xvfb
+
+# Used for running FF
+RUN apt-get install -y \
+    libnss3-tools \
+    libgtk-3-common \
+    libdbus-glib-1-2
+
+RUN mkdir /wptdashboard
+ADD . /wptdashboard
```

Comment:
I think our base image should only include dependency installation and possibly some general environment setup (i.e., exclude `ADD` here and `RUN` below). Could we have a `Dockerfile.jenkins` or something for this?

Reply:
What would the base image be used for other than test runs?

Reply:
To test our code, to run a dev server for compiling protos, and/or to deploy modules to other environments.

Reply:
Splitting into multiple Dockerfiles sounds reasonable, but for the sake of simplicity let's start with this one and as the scope is expanded with each PR we can split them up as necessary.